client/app/src/constants/Messages.js

```diff
@@ -0,0 +1,16 @@
+export const MESSAGE_INSTRUCTION_COLOR =
+  'Please indicate the color that you would primarily associate to the last 20 seconds of the piece.';
+export const MESSAGE_INSTRUCTION_EMOTION =
+  'Please evaluate the emotional intensity the last 20 seconds of the piece evoked in you.';
+
+export const MESSAGE_RESPONSE_MISSED_BODY = 'Please continue enjoying the music';
```

Comment:
Super nitpicky: would like to see a period added here to match end punctuation of other bodies.